### PR TITLE
Migration guide for new Material 3 `Slider` shapes

### PR DESCRIPTION
**Pull request by TahaTesser**
Related to https://github.com/flutter/flutter/pull/152237

## Presubmit checklist

- [x] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/content/release/breaking-changes/introduce-new-slider-shapes.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+description: >-
+  Adds new Material 3 `Slider` shapes which were introduced in the Material 3
+  specification.
+---
+
+## Summary
+
+The `Slider` widget now supports the new Material 3 visual style. This changes
+the default `Slider` shapes such as the track shape, thumb shape, and value
+indicator shape.
```
Comment (sfshaza2) [inline src/content/release/breaking-changes/introduce-new-slider-shapes.md:12]:
```suggestion
The `Slider` widget now supports the updated Material 3 visual style. This affects
the default `Slider` shapes, such as the track, thumb,, and value indicator shapes.
```

File: src/content/release/breaking-changes/introduce-new-slider-shapes.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Background
+
+The Material 3 specification introduces new `Slider` shapes. The new track
+shape includes an updated height, a gap between the active and inactive tracks,
+and stop indicators to show the end value of the inactive track. When the thumb
+is pressed, the thumb adjusts it's width and the track adjusts its shape. The
+new value indicator shape is a rounded rectangle. New color mappings are also
+introduced for some of the `Slider` shapes.
```
Comment (sfshaza2) [inline src/content/release/breaking-changes/introduce-new-slider-shapes.md:21]:
```suggestion
The Material 3 specification introduces updated `Slider` shapes. The new track
shape includes an updated height, a gap between the active and inactive tracks,
and stop indicators to show the end value of the inactive track. 
Pressing the thumb adjusts  its width and the track adjusts its shape. The
new value indicator shape is a rounded rectangle. New color mappings are also
introduced for some of the `Slider` shapes.
```

File: src/content/release/breaking-changes/introduce-new-slider-shapes.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+## Migration guide
+
+The `Slider` widget now supports the new Material 3 visual style. This changes
+the default `Slider` shapes such as the track shape, thumb shape, and value
+indicator shape.
```
Comment (sfshaza2) [inline src/content/release/breaking-changes/introduce-new-slider-shapes.md:27]: This was repeating the section above.

```suggestion
```

File: src/content/release/breaking-changes/introduce-new-slider-shapes.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+To use the previous `Slider` shapes and color mappings, set the following
```
Comment (sfshaza2) [inline src/content/release/breaking-changes/introduce-new-slider-shapes.md:29]:
```suggestion
You automatically get the new `Slider` shapes and color mappings.
To use the previous `Slider` shapes and color mappings, set the following
```

File: src/content/release/breaking-changes/introduce-new-slider-shapes.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  valueIndicatorColor: Theme.of(context).colorScheme.primary,
+  valueIndicatorTextStyle: Theme.of(context).textTheme.labelMedium!.copyWith(
+    color: Theme.of(context).colorScheme.onPrimary,
+  ),
+),
+```
+
+## Timeline
+
+Landed in version: TBA<br>
```
Comment (sfshaza2) [inline src/content/release/breaking-changes/introduce-new-slider-shapes.md:83]: Can you provide the version?

**Review by sfshaza2 — COMMENTED** [review]
Thanks for your PR! Can you provide more version info, such as which version on the main branch?

**Comment by Piinks** [comment]
@sfshaza2 the change has not landed yet, so no version tag. See the PR description.

**Comment by sfshaza2** [comment]
@Piinks, generally we land these PRs once they've at least hit main branch. But sure. :)

**Review by sfshaza2 — COMMENTED** [review]
lgtm

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Comment by Piinks** [comment]
We weren't trying to land this right now. We were getting feedback alongside the change. We need to revert this.

**Comment by Piinks** [comment]
> @Piinks, generally we land these PRs once they've at least hit main branch. But sure. :)

That has never been my experience. 🤷

**Comment by sfshaza2** [comment]
I'm really sorry, @TahaTesser, for giving you extra work. I messed up.

**Comment by TahaTesser** [comment]
> I'm really sorry, @TahaTesser, for giving you extra work. I messed up.

No worries!

**Pull request merged**